app.js

```diff
@@ -0,0 +1,59 @@
+// class Book {
+//   constructor (title, author) {
+//     this.title = title;
+//     this.author = author;
+//   }
+// }
+// const awesomeBooks = [];
+
+// awesomeBooks.push(new Book(title.value, author.value));
+
+// console.log(awesomeBooks);
+
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const listBooks = document.querySelector('.container ul');
+const myButton = document.querySelector('#btn-list');
+let index = 0;
+
+myButton.addEventListener('click', (e) => {
+  if (title.value !== '' && author.value !== '') {
+    e.preventDefault();
+    index += 1;
+
+    const titleBooks = document.createElement('li');
+    titleBooks.innerHTML = title.value;
+    listBooks.appendChild(titleBooks);
+
+    const authorBooks = document.createElement('li');
+    authorBooks.innerHTML = author.value;
+    listBooks.appendChild(authorBooks);
+
+    const remove = document.createElement('button');
+
+    // Event: Remove a Book
+    remove.innerText = 'Remove';
+    listBooks.appendChild(remove);
+    remove.setAttribute('id', `remove-${index}`);
+    const hr = document.createElement('hr');
+    listBooks.appendChild(hr);
+    const button = document.querySelector(`#remove-${index}`);
+    button.addEventListener('click', () => {
+      titleBooks.remove();
+      authorBooks.remove();
+      remove.remove();
+      hr.remove();
+    });
+  }
+});
+
+// Local storage
+const bookForm = document.querySelector('#addBook');
+const content = JSON.parse(window.localStorage.getItem('storage'));
+function check() {
+  if (localStorage.getItem('storage')) {
+    bookForm.elements.title.value = content.title;
+    bookForm.elements.author.value = content.author;
+  }
+}
```

Comment:
- [ ] You guys have done a great job in add and delete operations :+1: but i can see that you didn't implement the localStorage properly. :fearful: Let me explain you what we are expecting you to do :+1:

1. First when you will add your book then it should be added in the localStorage and then a render function will run which will get the latest data from localStorage and will show it on the browser.

2. Remove function is a bit different. You need to get the localStorage first and store it into a variable. After storing it into a variable you need to filter that book from the variable which you want to remove. After filtering the book you need to set the filtered data means the filtered variable to the localStoroage then your render function will run and get the latest data of localStorage and will render it :thinking: